### ISSUE TITLE
video_player anr

### ISSUE THREAD

**Issue by lihuange4** [flutter/flutter#67805]
## Steps to Reproduce

1. In android, video_player cause anr when call "exoPlayer.release()"

<details>
  <summary>Logs</summary>

"main" prio=5 tid=1 Waiting
  | group="main" sCount=1 dsCount=0 flags=1 obj=0x715e4a38 self=0xb400007757771380
  | sysTid=15711 nice=-10 cgrp=default sched=0/0 handle=0x78de55c4f8
  | state=S schedstat=( 1368840360 396685308 3420 ) utm=112 stm=23 core=7 HZ=100
  | stack=0x7ffe021000-0x7ffe023000 stackSize=8192KB
  | held mutexes=
  at java.lang.Object.wait(Native method)
  - waiting on <0x04a50072> (a com.google.android.exoplayer2.ExoPlayerImplInternal)
  at java.lang.Object.wait(Object.java:442)
  at java.lang.Object.wait(Object.java:568)
  at com.google.android.exoplayer2.ExoPlayerImplInternal.release(ExoPlayerImplInternal.java:226)
  - locked <0x04a50072> (a com.google.android.exoplayer2.ExoPlayerImplInternal)
  at com.google.android.exoplayer2.ExoPlayerImpl.release(ExoPlayerImpl.java:401)
  at com.google.android.exoplayer2.SimpleExoPlayer.release(SimpleExoPlayer.java:980)
  at io.flutter.plugins.videoplayer.VideoPlayer.dispose(VideoPlayer.java:278)
  at io.flutter.plugins.videoplayer.VideoPlayerPlugin.dispose(VideoPlayerPlugin.java:143)
  at io.flutter.plugins.videoplayer.Messages$VideoPlayerApi$3.onMessage(Messages.java:337)
  at io.flutter.plugin.common.BasicMessageChannel$IncomingMessageHandler.onMessage(BasicMessageChannel.java:187)
  at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
  at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:692)
  at android.os.MessageQueue.nativePollOnce(Native method)
  at android.os.MessageQueue.next(MessageQueue.java:335)
  at android.os.Looper.loop(Looper.java:193)
  at android.app.ActivityThread.main(ActivityThread.java:7887)
  at java.lang.reflect.Method.invoke(Native method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:656)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:967)

```
```

```
```

```
```

</details>

**Comment by iapicca** [comment]
@lihuange4 can you provide your `flutter doctor -v`, your `pubspec.yaml` and most importantly a minimal reproducible code sample

**Comment by darshankawar** [comment]
@lihuange4 
Please provide above requested details.
Thanks.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
